### ISSUE TITLE
Update README.md to mention that SELinux should be disabled

### ISSUE THREAD

**Issue by scherepanov** [neutrinolabs/xrdp#997]
All new distros are coming with SELinux enabled by default. 
This makes xrdp unusable.
Please modify README.md and mention that SELinux should be disabled, otherwise non-root users cannot connect to Xvnc, and Xorg does not work at all.
It would save a lot of time for me!

**Comment by metalefty** [comment]
I don't think SELinux should be disabled and it need not be disabled actually. Whether disabling SELinux or not is up to you but you shouldn't recommend disabling SELinux without thinking.

For example, Fedora has `xrdp-selinux` package. It will work fine if you define SELinux policy correctly.
http://pkgs.fedoraproject.org/cgit/rpms/xrdp.git/tree/xrdp.spec

**Comment by scherepanov** [comment]
Thanks for a hint - yes installation of selinux-xrdp makes xrdp usable with selinux.
That is not obvious and I would still suggest to mention it somewhere in README.

I am looking on out-of-box Fedora 27.
xrdp works with Xvnc OK, with or without SELinux, with addition of package xrdp-selinux. But, default Fedora 27 workstation install do enable SELinux and install xrdp, but no xrdp-selinux. That makes xrdp not usable on fedora 27 out of box, with no trivial diagnostics. Of course, thanks to your hint (again thanks for your time answering!) I can make it work. There are a lot of users in a wild who just see that xrdp does not work out of box and will not be able to fix.
Not to mention, xrdp works only with Xvnc (on Fedora 27) and failing with Xorg.
To make it work with Xorg, you need to create file /etc/X11/Xwrapper.config with content allowed_users=anybody. Again, it is not created by default install.
And even after you will create Xwrapper.config, Xorg is not going to work. It causes division by 0 in gsd-color in startup sequence, which then ANOM_ABEND. Desktop flashes and disappear.
All of the above is not xrdp fault. Except that most likely is not clear for average user. I think just mention about working configuration would have a lot of sense.
xrdp is vitally important product, thanks for working on it!!!

**Comment by metalefty** [comment]
Okay, the subject really you want is "give users more hints about working configuration", right? Disabling SELinux is a means but not the purpose. Let's solve them one by one.

> And even after you will create Xwrapper.config, Xorg is not going to work. It causes division by 0 in gsd-color in startup sequence, which then ANOM_ABEND.

This one is going to be fixed. neutrinolabs/xorgxrdp#111 should solve the division by 0 issue.

**Comment by scherepanov** [comment]
Yes, you correctly guessed what I really mean. I suggested to give users a hint about working configuration in README.
Nice to hear that problem with Xorg is in works.

**Comment by metalefty** [comment]
I've requested the maintainer of Fedora xorgxrdp package to apply that patch. I hope the division 0 issue solved for Fedora/EL xorgxrdp package.

**Comment by joseortiz3** [comment]
Any updates on a known working configuration?
